packages/node-api/integration/LoAuthorityList.ts

```diff
@@ -1,17 +1,20 @@
-import { ALICE, DAVE, FERDIE, setup, signAndSend } from "./Util.js";
+import { setup, signAndSend } from "./Util.js";
+import { encodeAddress } from "@polkadot/util-crypto";
 
 export async function addGuestLegalOfficer() {
     const { alice, api } = await setup();
-    const extrinsic = api.polkadot.tx.loAuthorityList.addLegalOfficer(DAVE, {
-        Guest: ALICE,
+    const dave = encodeAddress("5DAAnrj7VHTznn2AWBemMuyBwZWs6FNFjdyVXUeYum3PTXFy", api.queries.ss58Prefix);
```

Comment:
Could use `getValidAccountId` if applying my previous comment.

Reply:
Done in b8dee278